Using AWS secrets manager

yeseniamolinab opened the pull request.

subbiahf22 commented:
@yeseniamolinab I am not able to add you as reviewer, can you pls leave your review as well

yeseniamolinab commented:
@subbiahf22 I am addressing feedback from @osahyoun on https://github.com/SumOfUs/Champaign/pull/1956#discussion_r891249138

Let's wait for his reply and see if you need to also change something here 👍🏼

yeseniamolinab commented on app/lib/secrets_manager.rb:
```diff
@@ -0,0 +1,35 @@
+        secret_id: secret_name(secret_id)
+      ).secret_string)
+    rescue StandardError => e
+      logger.warn("Failed to get secret value for #{secret_name(secret_id)}!\n#{e}")
```
@subbiahf22 I changed this on the champaign's PR to just return nil.
You can leave it, but I think `logger` is not defined, if so, it should be `Rails.logger`
Could you please verify how we are calling the logger on this codebase?

yeseniamolinab reviewed (commented):
It looks good to me.
I don't see the config gem we have on champaign, so I think you're good at using the env variables unless @osahyoun believes otherwise.

I just have a comment about the logger (I think that line will break)

subbiahf22 commented:
@osahyoun Can you please have a look at this PR
